python/catkin/test_results.py

```diff
@@ -134,6 +135,12 @@ def test_results(test_results_dir):
                 print('Skipping "%s": %s' % (name, str(e)))
                 continue
             results[name] = (num_tests, num_errors, num_failures)
+            if show_verbose and (num_errors + num_failures > 0):
+                print("Test %s was failed" % (name))
+                print("-------------------------------------------------")
+                verbose = open(filename_abs, "r").read()
+                print(verbose)
+                print("-------------------------------------------------")
```

Comment:
thanks, fixed in https://github.com/k-okada/catkin/commit/cce91c1ee2f53ca8b40d3038781234d1d3e3b9f7